### ISSUE TITLE
Showing bounds of selection by coloring segments of ruler

### ISSUE THREAD

**Issue by Lehonti** [PintaProject/Pinta#1094]
When an area of is selected, the appropriate portion of the ruler could be highlighted with a different color.

**Comment by Lehonti** [comment]
@cameronwhite I thought of closing this, but I remembered that if there are multiple, disconnected selections, _the full range_ is highlighted, and not the separate ranges. This is by no means critical, but it might be nice to have

**Comment by cameronwhite** [comment]
Yeah I'd agree that it's a bit lower priority. I assume this would just go through the bounds of each selection polygon?

**Comment by Lehonti** [comment]
Yeah, it would go through the bounds of each selection polygon. But now that I think about it, it could be a separate issue
